<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QDialog" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1E1F2E;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>190</width>
     <height>165</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>0</y>
     <width>899</width>
     <height>199</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #252748;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>312</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#FFF;
border: none;
font-size: 55px;</string>
   </property>
   <property name="text">
    <string>HOME</string>
   </property>
  </widget>
  <widget class="QToolButton" name="Setting">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>90</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-style: normal;
font-weight: 900;
font-size: 35px;
line-height: 42px;
display: flex;
align-items: center;
text-align: center;
color:#FFF;
border: none;</string>
   </property>
   <property name="text">
    <string>Refund</string>
   </property>
  </widget>
  <widget class="QToolButton" name="LeaderBoard">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>90</y>
     <width>261</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 35px;
line-height: 42px;
display: flex;
align-items: center;
text-align: center;
border: none;
color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Leaderboards</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>899</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #f0f0f0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 54px;
line-height: 65px;
display: flex;
align-items: center;
text-align: center;
border:none;
color: #252748;
background: #f0f0f0;</string>
   </property>
   <property name="plainText">
    <string>GAMES</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
border: none;
color: #000000;
background: #f0f0f0
</string>
   </property>
   <property name="plainText">
    <string>FLAPPYBIRD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>324</width>
     <height>208</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>650</y>
     <width>326</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #BB5858;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;

color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>650</y>
     <width>371</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #BB5858;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;

color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>PLAY</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_3">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>340</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
border: none;
color: #000000;
background: #f0f0f0
</string>
   </property>
   <property name="plainText">
    <string>Hang Man GAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>430</y>
     <width>371</width>
     <height>208</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/C:/Users/ABC/Downloads/WhatsApp Image 2023-05-15 at 19.47.18.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Inter'; font-size:40px; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:28pt; font-weight:400;&quot;&gt;Username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="User_name">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
border:none;
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Inter'; font-size:40px; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:28pt; font-weight:400;&quot;&gt;Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;
border:none;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Inter'; font-size:40px; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:28pt; font-weight:400;&quot;&gt;Rank#:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;
border:none;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Inter'; font-size:40px; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:28pt; font-weight:400;&quot;&gt;Achievement:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;
border:none;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Inter'; font-size:40px; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:40px; font-weight:400;&quot;&gt;Score: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>630</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;</string>
   </property>
  </widget>
  <widget class="QLabel" name="screen2b">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>899</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #f0f0f0</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editprofile">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 54px;
line-height: 65px;
display: flex;
align-items: center;
text-align: center;

color: #252748;
background: #f0f0f0;
border:none;
</string>
   </property>
   <property name="plainText">
    <string>Edit Profile</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editname">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;

color: #252748;
background: #f0f0f0;
border:none;</string>
   </property>
   <property name="plainText">
    <string>Enter UserName:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editage">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;

color: #252748;
background: #f0f0f0;
border:none;</string>
   </property>
   <property name="plainText">
    <string>Enter Name:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editpassword">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>540</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;

color: #252748;
background: #f0f0f0;
border:none;</string>
   </property>
   <property name="plainText">
    <string>Enter Complain:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Refund_complain">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>550</y>
     <width>397</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
border_radius:30px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Refund_Name">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>450</y>
     <width>397</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
border_radius:30px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Refund_userName">
   <property name="geometry">
    <rect>
     <x>856</x>
     <y>350</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;
background: #1E1F2E;
color: #FFFFFF;
border_radius:30px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="screen2a">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>899</width>
     <height>199</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #252748;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="Settings2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>312</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 55px;
line-height: 82px;
display: flex;
align-items: center;
text-align: center;
text-decoration-line: underline;
border: none;
color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Refund</string>
   </property>
  </widget>
  <widget class="QToolButton" name="Home2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>80</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-style: normal;
font-weight: 900;
font-size: 35px;
line-height: 42px;
display: flex;
align-items: center;
text-align: center;
color:#FFF;
border: none;</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QToolButton" name="noti2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>80</y>
     <width>281</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 35px;
line-height: 42px;
display: flex;
align-items: center;
text-align: center;
border: none;
color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Leaderboards</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>230</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Refrsh Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>650</y>
     <width>326</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #BB5858;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 40px;
line-height: 48px;
display: flex;
align-items: center;
text-align: center;

color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>260</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <zorder>pushButton_4</zorder>
  <zorder>editpassword</zorder>
  <zorder>editname</zorder>
  <zorder>editage</zorder>
  <zorder>screen2b</zorder>
  <zorder>Refund_Name</zorder>
  <zorder>Refund_userName</zorder>
  <zorder>Refund_complain</zorder>
  <zorder>editprofile</zorder>
  <zorder>screen2a</zorder>
  <zorder>Settings2</zorder>
  <zorder>Home2</zorder>
  <zorder>noti2</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>toolButton</zorder>
  <zorder>Setting</zorder>
  <zorder>LeaderBoard</zorder>
  <zorder>label_4</zorder>
  <zorder>plainTextEdit</zorder>
  <zorder>plainTextEdit_2</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>plainTextEdit_3</zorder>
  <zorder>label_6</zorder>
  <zorder>textEdit</zorder>
  <zorder>User_name</zorder>
  <zorder>textEdit_2</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>textEdit_3</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>textEdit_4</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>textEdit_5</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
